apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-maximum/get-api-rate-limit-maximum.usecase.ts

```diff
@@ -24,9 +24,9 @@ export class GetApiRateLimit {
   }
 
   @CachedEntity({
-    builder: (command: GetApiRateLimitCommand) =>
+    builder: (command: { apiRateLimitCategory: ApiRateLimitCategoryEnum; _environmentId: string }) =>
       buildMaximumApiRateLimitKey({
-        _environmentId: command.environmentId,
+        _environmentId: command._environmentId,
```

Comment:
sorry about that, i still have not found a way to do the cache decorator type safe. the input signature in the builder should be the same as the wrapped function input.